### ISSUE TITLE
Passing id/slug to the URL

### ISSUE THREAD

**Issue by alaminopu** [motiv-labs/janus#248]
How should I pass `<id>` or `<slug>` to a specific URL. I was trying like the following, but it doesn't work. 

```
{
    "name" : "example",
    "active" : true,
    "proxy" : {
        "preserve_host" : false,
        "listen_path" : "/example/{id}",
        "upstreams" : {
            "balancing": "roundrobin",
            "targets": [
                {"target": "https://jsonplaceholder.typicode.com/posts/{id}"}
            ]
        },
        "strip_path" : false,
        "append_path" : false,
        "methods" : ["GET"]
    },
    "health_check": {
        "url": "http://service1:8080/status",
        "timeout": 3
    }
}
```

**Comment by italolelis** [comment]
Hi @alaminopu,

This feature is not supported at the moment. We use [chi](https://github.com/go-chi/chi) as our router and would have to change the transport layer to parse this kind of rules. 
The example that you've shown can be easily done by having a config like this:

```
{
    "name" : "example",
    "active" : true,
    "proxy" : {
        "preserve_host" : false,
        "listen_path" : "/example/*",
        "upstreams" : {
            "balancing": "roundrobin",
            "targets": [
                {"target": "https://jsonplaceholder.typicode.com/posts"}
            ]
        },
        "strip_path" : true,
        "append_path" : false,
        "methods" : ["GET"]
    },
    "health_check": {
        "url": "http://service1:8080/status",
        "timeout": 3
    }
}
```

We can think about implementing such a feature, but at the moment we are very busy and have no time to work on this. If you'd like to help we would be very happy.

Thanks

**Comment by italolelis** [comment]
Solved by #262
